.github/workflows/test.yml.erb

```
@@ -0,0 +1,31 @@
+name: Unit tests
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ruby: "2.5"
+          - ruby: "2.6"
+          - ruby: "2.7"
+          - ruby: "3.0"
+            coverage: "yes"
+    env:
+      COVERAGE: ${{ matrix.coverage }}
```

This was set up specifically in combination with `Gemfile` and rake tasks. Here that isn't present so you can simplify it.
```suggestion
        ruby:
          - "2.5"
          - "2.6"
          - "2.7"
          - "3.0"
```